dw-select-item.js

```diff
@@ -158,18 +170,35 @@ export class DwSelectItem extends LitElement {
     this.selected = false;
   }
 
+  update(changedProps) {
+    if (changedProps.has('item')) {
+      this._title = this._getName(this.item, this.itemLabel);
+    }
+    super.update(changedProps);
+  }
+
+  updated(changedProps) {
+    super.updated(changedProps);
+    if (changedProps.has('_title') || changedProps.has('disabledTooltip')) {
+      this._setTooltipText();
+    }
+  }
+
   render() {
     return html`
-      <div class="container" title=${this._getToolTipText()}>
+      <div class="container">
         <div class="icon" ?hidden="${!this.icon}">
           <dw-icon style="${styleMap(this._setIconColor(this.item))}" name="${this.icon}" size="${this.iconSize}"></dw-icon>
         </div>
-        <div class="content" style="${styleMap(this._setTextColor(this.item))}">${this._getName(this.item, this.itemLabel)}</div>
-        <div class="check-icon">${this.selected ? this._getCheckIcon() : ''}</div>
+        <div class="content" style="${styleMap(this._setTextColor(this.item))}">${this._title}</div>
```

Comment:
@nirmalbaldaniya-dream this component is also being used by the Athena. So, we should make sure, it isn't causing any regression (change in behavior) to them.